snapshotter/demux/proxy/snapshotter.go

```diff
@@ -73,6 +73,14 @@ func NewRemoteSnapshotter(ctx context.Context, address string,
 	return &RemoteSnapshotter{proxy.NewSnapshotter(snapshotsapi.NewSnapshotsClient(gRPCConn), address), metricsProxy}, nil
 }
 
+// Cleanup implements the Cleaner interface for snapshotters.
+// This enables asynchronous resource cleanup by remote snapshotters.
+//
+// See https://www.github.com/containerd/containerd/blob/release/1.6/snapshots/snapshotter.go
```

Comment:
How about using https://github.com/containerd/containerd/tree/v1.6.4? release/1.6 wouldn't change much but in theory moves still.

Reply:
Ah agreed. I had the concepts in my head switched with respect to how they moved. Thanks for clarification.